src/cdev/commands/initializer.py

```diff
@@ -6,15 +6,16 @@
 from typing import Dict, Tuple, List
 
 from rich.prompt import Prompt
-from cdev.default.project import local_project
+from cdev.default.project import local_project, local_project_info
 from cdev.cli.logger import set_global_logger_from_cli
 
 from core.constructs.backend import Backend, Backend_Configuration
 from core.constructs.workspace import Workspace_Info
 from core.default.backend import Local_Backend_Configuration, LocalBackend
+from core.utils import paths as paths_util
 
 
-from ..constructs.project import Project_State, check_if_project_exists, project_info
+from cdev.constructs.project import Project_State, check_if_project_exists, Project_Info
```

Comment:
normally on python we declare the models/classes with a capital letter in the beginning of each word, in this case it would be ProjectInfo.
But that is not a big deal
https://realpython.com/python-pep8/